proto/src/constants.rs

```diff
@@ -248,6 +248,7 @@ pub const SUB_ATTR_TYPE: &str = "type";
 pub const SUB_ATTR_VALUE: &str = "value";
 
 pub const OAUTH2_SCOPE_EMAIL: &str = ATTR_EMAIL;
+pub const OAUTH2_SCOPE_UPDATED_AT: &str = "updated_at";
```

Comment:
fyi this field is part of the `profile` scope - https://openid.net/specs/openid-connect-core-1_0.html#ScopeClaims there isn't a separate one for each claim (how to request specific claims directly is described below, in 5.5)

Reply:
True, and i know that i have read that.. not sure why i have implemented a separate one..
Probably overmotivation :smile:  I'll change that